tasks/rsync-deploy.yml

```diff
@@ -1,4 +1,16 @@
 ---
+
+# Migration check from symlink deployment
+- stat:
+    path: "{{ ansistrano_deploy_to }}/{{ ansistrano_current_dir }}"
+  register: stat_current_dir
+
```

Comment:
See #115 for a explanations why there is a special symlink handling is needed for rsync deployment.